core/example/benchmark/CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
+
 if(Cabana_ENABLE_Serial)
   add_executable(NeighborListMDPerfTest md_neighbor_perf_test.cpp)
```

Comment:
In the future I would like this to be in a performance test as well. Basically, everything in `example/benchmark/` should be a performance test eventually.

Reply:
fixed